internal/executor/exec.go

```diff
@@ -51,7 +51,7 @@ func Build(packagePath, outputFile string) (string, error) {
 	cmd := exec.Command(
 		"go",
 		"test",
-		"-gcflags=-N -l",  // disable optimization
+		`-gcflags="-N -l"`,  // disable optimization
```

Comment:
Please use this instead: `-gcflags=all=-N -l`,

Reply:
I get the same error as it seems those flags are passed to `go` instead of the `gc`
```shell
error walking the path: failed to build test file: failed to execute build command '/home/linuxbrew/.linuxbrew/bin/go test -gcflags=all=-N -l -c ./libbpfgo/helpers -o __libbpfgo_helpers.test': exit status 1
```

like below:
```shell
$ go test -gcflags=all=-N -l -c ./libbpfgo/helpers -o __libbpfgo_helpers.test
go: unknown flag -l cannot be used with -c
usage: go test [build/test flags] [packages] [build/test flags & test binary flags]
Run 'go help test' and 'go help testflag' for details.
```

```shell
$ go version
go version go1.23.3 linux/amd64
```